packages/usa-button/src/usa-button.twig

```diff
@@ -2,20 +2,80 @@
   <div class="bg-base-darkest padding-1" style="max-width: fit-content">
 {% endif %}
 
+{% set icon %}
+  <!-- @TODO: Allow size to be passed as a argument. -->
+  <!-- usa-icon--size-{3-9} -->
+  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
+    <use xlink:href="./img/sprite.svg#{{ icon_name | default("add_circle_outline") }}"></use>
+  </svg>
+{% endset %}
+
 {% if is_demo and 'usa-button--big' not in modifier %}
-  <button type="{{ type }}" class="usa-button {{ modifier }}">Default</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--hover">Hover</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--active">Active</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }} usa-focus">Focus</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }}" disabled>Disabled</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }}" aria-disabled="true">aria-disabled</button>
-  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--unstyled">Unstyled button</button>
+  <button type="{{ type }}" class="usa-button {{ modifier }}">
+    Default
+    {% if icon_included %}
+    {{ icon }}
+    {% endif %}
+  </button>
+  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--hover">
+    Hover
+    {% if icon_included %}
+    {{ icon }}
+    {% endif %}
+  </button>
+  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--active">
+    Active
+    {% if icon_included %}
+    {{ icon }}
+    {% endif %}
+  </button>
+  <button type="{{ type }}" class="usa-button {{ modifier }} usa-focus">
+    Focus
+    {% if icon_included %}
+    {{ icon }}
+    {% endif %}
+  </button>
+  <button type="{{ type }}" class="usa-button {{ modifier }}" disabled>
+    Disabled
+    {% if icon_included %}
+    {{ icon }}
+    {% endif %}
+  </button>
+  <button type="{{ type }}" class="usa-button {{ modifier }}" aria-disabled="true">
+    aria-disabled
+    {% if icon_included %}
+    {{ icon }}
+    {% endif %}
+  </button>
+  <button type="{{ type }}" class="usa-button {{ modifier }} usa-button--unstyled">
+    Unstyled button
+    {% if icon_included %}
+    {{ icon }}
+    {% endif %}
+  </button>
 {% else %}
-  <button type="{{ type }}" class="usa-button {{ modifier }}">{{ text }}</button>
+  <button type="{{ type }}" class="usa-button {{ modifier }}">
+    {{ text }}
+  </button>
```

Comment:
I'd be interested in being able to see the icon even when `is_demo` is false.

![image](https://github.com/uswds/uswds/assets/25211650/a4d48126-6bab-4082-8d70-f8f9b420a828)


```suggestion
  <button type="{{ type }}" class="usa-button {{ modifier }}">
    {{ text }}
    {% if icon_included %}
    {{ icon }}
    {% endif %}
  </button>
```

Reply:
Good point. Created a macro for more consistent display in 87fc766. 


_We still need #5394 to clean this up further._